### ISSUE TITLE
Playback failed: one or more items failed to play, check the log for more information

### ISSUE THREAD

**Issue by CXgamer** [add-ons/plugin.video.vtm.go#196]
**Describe the bug**
When attempting to launch an item from the VTM GO catalog, playback fails, prompting to check the log.

**To Reproduce**
1. Open the VTM GO addon
2. A-Z
3. Select any item
4. Observe the error

**Expected behavior**
Instead of the error, the playback should have started.

**System**
 - Addon version: 1.1.0
 - Kodi version: 18.5
 - Inputstream adaptive version: 2.4.5
 - Operating System: Windows 10
 - Special Hardware: Desktop

**Additional context**
[kodi.log](https://github.com/add-ons/plugin.video.vtm.go/files/5079424/kodi.log)

**Comment by michaelarnauts** [comment]
Interesting. Is it really with all items? It seems to fail when processing a subtitle with a special character, so I assume it's not with all items.

**Comment by CXgamer** [comment]
Yes, we tried with several shows of the same series and one from a different show.

**Comment by mediaminister** [comment]
Can you test this fix: 
[plugin.video.vtm.go-1.1.0-listdir-5ab1cfa.zip](https://github.com/add-ons/plugin.video.vtm.go/files/5080508/plugin.video.vtm.go-1.1.0-listdir-5ab1cfa.zip)
